src/WorkOS.net/Services/MFA/Entities/Factor.cs

```diff
@@ -0,0 +1,47 @@
+namespace WorkOS
+{
+    using System;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Contains information about a MFA Factor record.
```

Comment:
nit: We should be using `an` as the article before `MFA` as it agrees with the pronunciation: "em-eff-ay"

```suggestion
    /// Contains information about an MFA Factor record.
```